### DISCUSSION TITLE
How to Load and Evaluate a Local HF-Formatted Model.

### DISCUSSION THREAD

**Discussion by liu904-61** [open-compass/opencompass#605] [Q&A] [answered]
A model based on LLAMA2

**Comment by Leymore** [comment, marked as answer]
Set the `path` and `tokenizer_path` variables to your local path

https://github.com/open-compass/opencompass/blob/main/configs/models/hf_llama/hf_llama2_7b.py#L8-L9

**Reply by liu904-61** [reply]
ok，thinks。

**Reply by liu904-61** [reply]
Sorry, additional question: How should the 5shot evaluation be conducted?
